presto-docs/src/main/sphinx/release/release-0.209.rst

```diff
@@ -0,0 +1,78 @@
+=============
+Release 0.209
+=============
+
+General Changes
+---------------
+
+* Fix incorrect predicate pushdown when grouping sets contain the empty grouping set (:issue:`11296`).
+* Fix ``X-Forwarded-Proto`` header handling for requests to the ``/`` path (:issue:`11168`).
+* Fix a regression that results in execution failure when at least one
+  of the arguments to :func:`min_by` or :func:`max_by` is a constant null.
+* Fix execution failure of queries using bucket-by-bucket execution when
+  some buckets are completely filtered out.
+* Fix execution failure of queries due to a planning deficiency involving
+  complex nested joins where a join that is not eligible for bucket-by-bucket
+  execution feeds into the build side of a join that is eligible.
+* Improve numerical stability for :func:`corr`, :func:`covar_samp`,
+  :func:`regr_intercept`, and :func:`regr_slope`.
+* Do not include column aliases when checking column access permissions.
+* Eliminate unnecessary data redistribution for scalar correlated subqueries.
+* Remove table scan original constraint information from ``EXPLAIN`` output.
+* Change exceeded memory limit errors to have distinct error codes for global and per-node limits.
```

Comment:
Introduce distinct error codes for global and per-node "memory limit exceeded" errors.